core/testing/src/main/java/com/watcha/testing/util/MainDispatcherRule.kt

```diff
@@ -0,0 +1,19 @@
+package com.watcha.testing.util
+
```

Comment:
```suggestion
package com.terning.testing.util

```
여기 패키지도 수정 부탁드려요...!!

Reply:
아 패키지 자체 네이밍이 잘못되어 있었네요 진짜진짜 클날뻔 ㅜ ㅜ 다시 만들었습니댜